src/librustc_mir/const_eval/machine.rs

```diff
@@ -56,7 +56,7 @@ impl<'mir, 'tcx> InterpCx<'mir, 'tcx, CompileTimeInterpreter<'mir, 'tcx>> {
         self.copy_op(place.into(), dest)?;
 
         self.return_to_block(ret.map(|r| r.1))?;
-        self.dump_place(*dest);
+        trace!("{:?}", self.dump_place(*dest));
```

Comment:
Btw, why are these `Debug` impls? They are definitely made for human consumption, so sounds more like a case of `Display` to me TBH... (but changing it now is probably not worth it because Miri also needs adjusting).